### PR TITLE
Setup database connection

### PR DESCRIPTION
**Pull request by Melo5154**

### CODE REVIEW

File: gdsc-web-backend/Database/AppDbContext.cs [hunk 1]
```diff
@@ -0,0 +1,11 @@
+using Microsoft.EntityFrameworkCore;
+
+namespace gdsc_web_backend.Database
+{
+    public class AppDbContext :DbContext
```
Comment (danpercic86) [inline gdsc-web-backend/Database/AppDbContext.cs:5]: You've forgotten a space after colon =)))

Comment (Melo5154) [reply gdsc-web-backend/Database/AppDbContext.cs:5]: Am rezolvat :)

**Comment by danpercic86** [comment]
We need to merge this branch into dev first :(

**Comment by Melo5154** [comment]
Hope this time its all good

**Comment by danpercic86** [comment]
> Hope this time its all good

Well, it would be better to undo merge into main, since we need to merge it first into dev using this pull request, then we'll merge dev into main, to be sure that on main will be no errors

**Comment by Melo5154** [comment]
i did a revert there isn't that supposed to get in a previous verion?

**Comment by danpercic86** [comment]
> i did a revert there isn't that supposed to get in a previous verion?

I can't see any revert commit there, please check if you pushed you changes. Also, I'd suggest to just do a hard reset of `Setup-database-connection` branch into `Adaugat framework+prima migrare` commit. This way, the commit graph is cleaner

**Comment by Melo5154** [comment]
m8 i need some help ! i kinda fucked it all (main and my branch )and i need a secound oppinion to unfuck the project

**Pull request merged**